setup.py

```diff
@@ -3,7 +3,7 @@
 # or saved.  Do not modify them directly here.
 # NB: PACKAGES is deprecated
 NAME = "ZenPacks.zenoss.Import4"
-VERSION = "0.0.9dev"
+VERSION = "0.0.9"
```

Comment:
I noticed that dev was not properly processed by setup.py and it change the version to 0.0.9dev to 0.0.9dev0.. any idea why?